### PR TITLE
Add min-width & min-height to pagination links

### LINKED ISSUE #533: Pagination links: the clickable area is too small
**Describe the bug**
The clickable area of the various pagination links is a bit too small for a theme that aims to be WCAG level AAA "ready".

The posts pagination links have a target size of approximately 13 by 42 pixels:

posts pagination

where the width is a bit too small for all users.

Similarly, the links of a paginated post have a clickable area that is too small:

paginated post

Worth reminding that the WCAG Target Size (level AAA) success criterium requires a target size of 44 by 44 pixels, with some exceptions: https://www.w3.org/TR/WCAG21/#target-size

### PR DESCRIPTION
**Pull request by aristath**
Fixes #533

## Summary
Make sure the clickable area is at least 44x44px. 

## Quality assurance
* [x] I have thought about any security implications this code might add.
* [x] I have checked that this code doesn't impact performance (greatly).
* [x] I have tested this code to the best of my abilities
* [x] I have checked that this code does not affect the accessibility negatively

### CODE REVIEW

**Review by melchoyce — APPROVED** [review]
LGTM 👍

**Comment by carolinan** [comment]
This improvement needs to be added to the columns pagination and content pagination (.page-links for wp_link_pages() ).

**Comment by aristath** [comment]
> This improvement needs to be added to the columns pagination and content pagination (.page-links for wp_link_pages() ).

Done.
These used to have a vertical padding which made them 42.8px-tall.
The padding was removed and replaced with max-height & max-width.

**Comment by carolinan** [comment]
Sorry for the fatal typo here: ` columns pagination` 🤣 🤦  , I have confirmed that the comment pagination is also the correct size.

File: assets/sass/06-components/pagination.scss [hunk 1]
```diff
@@ -25,6 +25,11 @@
 
 	.nav-links {
 
+		> * {
```
Comment (afercia) [inline assets/sass/06-components/pagination.scss:28]: Just a quick question: why the direct children + universal selector pattern `> *` is used so widely across the theme's CSS? Not sure in this case there's the need to use it. Seems to me all the targeted elements do have a specific CSS class.

Comment (afercia) [reply assets/sass/06-components/pagination.scss:28]: Update: quickly discussed this on Slack, agreed to consider later refinements. Not a blocker.

**Pull request merged**